src/functions/pollTestCentreExaminers/framework/repo/mysql/test-centre-row-mapper.ts

```diff
@@ -11,7 +11,7 @@ export const buildTestCentreRowsFromQueryResult = (
   ));
 };
 
-const mapStaffNumber = (row: TestCentreRow): string => row.STAFF_NUMBER;
+const mapStaffNumber = (row: TestCentreRow): number => +row.STAFF_NUMBER;
```

Comment:
There is a function inside the repo i.e. `trimLeadingZeroes` (src/functions/pollUsers/framework/repo/mysql/examiner-record-row-mapper.ts) which can utilised here and then the rest of the logic can remain as is

This would then become:
`const mapStaffNumber = (row: TestCentreRow): string => trimLeadingZeroes(row.STAFF_NUMBER);`